cli/src/semgrep/engine.py

```diff
@@ -63,7 +63,8 @@ def get_cpu_count() -> int:
     def default_jobs(self) -> int:
         if self == EngineType.PRO_INTERFILE:
             return 1
-        return self.get_cpu_count()
+        # Maxing out number of cores used to 16 if more not requested to not overload on large machines
```

Comment:
If I didn't know what happened here, and I just found this comment, I would not know what "overload" means here.